silta-php-fpm/8.2-fpm/Dockerfile

```diff
@@ -119,6 +119,9 @@ COPY config/w-php-defaults.ini /usr/local/etc/php/conf.d/w-php-defaults.ini
 COPY silta /silta
 COPY silta/entrypoints/10-env.sh /etc/profile.d/10-env.sh
 
+# Elasticdump for importing/exporting Elasticsearch data
+RUN apk add npm nodejs && npm install -g elasticdump
```

Comment:
Please move this line a bit further up, above lines 95-96 which do cleanup. Also, add `&& npm cache clean --force` after the npm install command.

Reply:
was under impression npm is still required. 
code adjusted, works
```bash
$ elasticdump --version
6.111.0
```

Reply:
Hmm @Jancis I don't see any changes in the code

Reply:
i don't know where they went, recreated :)